Userland/Applications/Browser/Tab.cpp

```diff
@@ -92,8 +92,36 @@ Tab::Tab(BrowserWindow& window, Type type)
     else
         m_web_content_view = webview_container.add<Web::OutOfProcessWebView>();
 
-    toolbar.add_action(window.go_back_action());
-    toolbar.add_action(window.go_forward_action());
+    auto& go_back_button = toolbar.add_action(window.go_back_action());
+    go_back_button.on_context_menu_request = [this](auto& context_menu_event) {
+        if (m_history.can_go_back()) {
```

Comment:
let's invert the logic here, less nesting required.

```suggestion
        if (!m_history.can_go_back())
            return;
        ...
```